src/ucp/tag/eager.h

```diff
@@ -87,8 +87,8 @@ void ucp_tag_eager_sync_zcopy_req_complete(ucp_request_t *req, ucs_status_t stat
 void ucp_tag_eager_sync_zcopy_completion(uct_completion_t *self);
 
 static UCS_F_ALWAYS_INLINE int
-ucp_proto_eager_check_op_id(const ucp_proto_init_params_t *init_params,
-                            ucp_operation_id_t op_id, int offload_enabled)
+ucp_proto_tag_eager_check_op_id(const ucp_proto_init_params_t *init_params,
```

Comment:
Could it be  `ucp_tag_eager_check_op_id` ?

Reply:
why would it better? I'd keep `proto` in the name to highlight it is used by proto v2 only

Reply:
ok, but this is the existing convention, so we have to either follow it or consider renaming existing names
@yosefe, wdyt?

Reply:
In general, I agree with @rakhmets ; we should use filename-consistent naming convention.
However in the time being IMO it's more important to distinguish new and old protocols. So far we used "ucp_proto_XX" prefix for everything related to new protocols. After we finish the transition, we would be free to rename according to file/directory hierarchy.

Reply:
I'd rather use separate files.
E.g. all new AM protocols are in `src/ucp/am `.
I think using `ucp_proto_` fits for common functions, infrastructure, etc. But we can avoid using of it for protocol implementations. So there is no need to update names in the future.